src/js/csvManager.js

```diff
@@ -0,0 +1,123 @@
+const { sendFormToDatabase } = require('./databaseConnector');
+const { getFormsFromDatabase } = require('./databaseConnector');
+const { $id } = require('./utils');
+const Dialogs = require('./common/dialogs');
+
+exports.saveCsv = function (data) {
+    let readForm = [];
+    let readAnswers = [];
+    let csvOutput = '';
+    readForm = data.questions;
+    for (let i = 0; i < readForm.length; i++) {
+        let csvStringOneLine = '';
+        csvStringOneLine =
+      `"${
+          readForm[i].number
+      };${
+          readForm[i].type
+      };${
+          readForm[i].language
+      };${
+          readForm[i].content
+      };${
+          readForm[i].numberOfAnswers
+      };`;
+        readAnswers = readForm[i].answers;
+        for (let j = 0; j < readAnswers.length; j++) {
+            csvStringOneLine += `${readAnswers[j]};`;
+        }
+        csvOutput += `${csvStringOneLine}";;;;;;` + '\r\n';
```

Comment:
ujednolicić tego stringa. Po to się sotsuje `` żeby nie dodawać. (To \r\n można wrzucić bezpośrednio przed ostatnim `)